lib/go-tc/deliveryservice_servers.go

```diff
@@ -78,3 +106,63 @@ type DssServer struct {
 	TypeID           *int                 `json:"typeId" db:"server_type_id"`
 	UpdPending       *bool                `json:"updPending" db:"upd_pending"`
 }
+
+// DSSDeliveryService - a version of the deliveryservice that allows for all fields to be null
```

Comment:
GoDoc comments should be complete sentences, i.e. "DSSDeliveryService is a version of..."